tristan/ding-wall/src/components/ListSection.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 import './ListSection.css';
+import ListTasks from './ListTasks';
+// import InputForm from './InputForm';
 
+/* This file seems redundant now, since InputForm is now a child of 
+ListTasks, though that was not what I set out to do.
+*/
```

Comment:
You can move the lines:
```jsx
  const initialTasks = [{text: "buy lumber"},
                        {text: "fix the house"},
                        {text: "move in"}];

  const [tasks, setTasks] = useState(initialTasks);  

  const addTask = text => {
    setTasks([...tasks, { text }]);
  };
```

Into this file, so the state is managed here, and just pass `tasks` into ListTasks as a prop. You will need to restructure ListTasks to map the tasks like ```props.tasks.map```.

Then you can move `<InputForm addTask={addTask}/>` back here, and bob's your uncle.

In fact, you probably should do this.